src/views/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -48,7 +48,8 @@
     if (
       !(
         e.target instanceof HTMLElement &&
-        e.target.hasAttribute("data-line-type")
+        (e.target.hasAttribute("data-line-type") ||
+          e.target.classList.contains("diff-line-content"))
```

Comment:
clicking on anything on the page, causes the highlighted part to be de-selected, so I make an exception for `diff-line-content` class here.

it's still not 100% accurate, and I guess it has do to with tables cell, but it's good enough.